tools/cli/commands-packages.js

```diff
@@ -1320,7 +1314,10 @@ main.registerCommand({
     });
 
     if (showJson) {
-      Console.info(JSON.stringify(jsonOut))
+      // we can't use Console here, because it pretty prints the output with
+      // a wrap at 80 chars per line, which causes the json to break if details
+      // options is active and the package descriptions exceed the limit
+      console.info(JSON.stringify(jsonOut));
```

Comment:
You can use `Console.noWrap` to not wrap the text: https://github.com/meteor/meteor/blob/5b5114979d05d0ce7cc5010bb68fd16725de35a3/tools/console/console.js#L23-L30